README.md

```diff
@@ -4,13 +4,14 @@ Code to run on every board, to run canary tests
 ## To set up a new board:
 - Create a new robot, likely in the `viam-hardware-testing` org, in the "Board Canaries" location.
 - Add a board to that robot, whose type matches your physical hardware.
-- In the `Auth/Network` tab of the config set the bind address to port 9090.
+- In the `CONFIGURE -> Builder` tab click the `+` sign and select `Network`. Then set the bind address to port 9090 in `Network`.
 - Install Viam on the physical board, per the Setup tab in the robot you just created. **However,** put the config file in `/etc/viam-canary.json` instead of `/etc/viam.json`.
 - Jump together 2 pins on your board. Ideally, one of them (the "output" pin) will also have hardware PWM enabled.
   - If you have a board whose GPIO pins cannot also do hardware PWM but you want to test hardware PWM separately, jump together a second set of pins. One set will be to test GPIO input and output, and the other will test hardware PWM output and digital interrupt input.
   - Also jump together 2 more gpio pins to test software PWM.
 - Clone this repo onto the board.
 - Run `sudo ./install.sh`
+  - if you are getting `error: externally-managed-environment` while running the command, then you need to setup `venv` in the directory. 
 - Follow the instructions printed at the end of that script:
```

Comment:
Add a mention of monitor_config.py on the monitor board. When that was new, I added instructions to install.sh but didn't think to update the readme to match.

Reply:
...and if we're talking about other things that need to happen for a board canary, put in a line about updating the "Team Hardware" page in the wiki to have login info.

Reply:
Nice! I'll add them.